application/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1 @@
+Automatic-Module-Name: io.vertx.core.application
```

Comment:
as said somewhere else, this cannot be `io.vertx.core.XXX` because we do have classes in `io.vertx.core` and it is already a module by itself. Module names have to be disjoint.